pkg/virtual/framework/forwardingregistry/store.go

```diff
@@ -273,3 +290,29 @@ func clientGetter(dynamicClusterClient dynamic.ClusterInterface, namespaceScoped
 		}
 	}
 }
+
+// updateToCreateOptions creates a CreateOptions with the same field values as the provided PatchOptions.
+func updateToCreateOptions(uo *metav1.UpdateOptions) metav1.CreateOptions {
+	co := metav1.CreateOptions{
+		DryRun:          uo.DryRun,
+		FieldManager:    uo.FieldManager,
+		FieldValidation: uo.FieldValidation,
+	}
```

Comment:
this will bite us when UpdateOptiosn is extended. How can we catch that case, e.g. in unit tests. Even some reflection based check would be fine that UpdateOptions has not gotten new fields.